bundler.d/assets.rb

```diff
@@ -1,9 +1,9 @@
 group :assets do
   gem 'jquery-turbolinks', '~> 2.1'
-  gem 'jquery-ui-rails', '< 5.0.0'
-  gem 'patternfly-sass', '>= 3.32.1', '< 3.38.0'
+  gem 'jquery-ui-rails', '~> 5.0'
+  gem 'patternfly-sass', '~> 3.59'
   gem 'gettext_i18n_rails_js', '~> 1.0'
   gem 'execjs', '>= 1.4.0', '< 3.0'
   gem 'uglifier', '>= 1.0.3'
-  gem 'sass-rails', '>= 5.0', ((RUBY_VERSION < '2.4') ? '< 5.0.8' : '< 6.0')
+  gem 'sass-rails', '~> 6.0'
```

Comment:
This becomes wrapper for `sassc-rails` - for some reason we can't use `sassc-rails` directly :man_shrugging:

Reply:
Related: https://github.com/theforeman/rails-packaging/pull/153 - that update could got into develop unrelated to the sprockets update, but we need to wait until the RPM repos are merged.